Firmware/ultralcd.cpp

```diff
@@ -1824,7 +1824,7 @@ static void lcd_support_menu()
     
   MENU_ITEM(submenu, MSG_MENU_TEMPERATURES, lcd_menu_temperatures);
 
-#if defined VOLT_BED_PIN || defined VOLT_BED_PIN
+#if defined (VOLT_BED_PIN) || defined (VOLT_BED_PIN)
```

Comment:
This OR makes no sense. Perhaps here is meant VOLT_PWR_PIN? Just guessing from line 1698

Reply:
Yes, you are right :-) Thanks.